pkgs/top-level/python-packages.nix

```diff
@@ -19123,6 +19123,29 @@ let
     };
   };
 
+  cmdtest = buildPythonPackage rec {
+    name = "cmdtest-${version}";
+    version = "0.15";
+    disabled = isPy3k || isPyPy;
+
+    buildInputs = with self; [ cliapp ttystatus ];
+    propagatedBuildInputs = with self; [ cliapp ttystatus markdown ];
+
+    # TODO: cmdtest tests must be run before the buildPhase
```

Comment:
In addition I've just noticed that the test executes cmdtest within the builddir, where cmdtest will assume python is present at /usr/bin/python, so this likely needs to be fixed also, potentially by simply having the test invoke python on cmdtest directly rather than relying on the hashbang.